resources/lang/en/follows.php

```diff
@@ -4,6 +4,18 @@
 // See the LICENCE file in the repository root for full licence text.
 
 return [
+    'forum_topic' => [
+        'title' => 'forum topic',
```

Comment:
I think naming these after what their sections are named instead of the code makes more sense.

Reply:
what do you mean?

Reply:
topics/beatmap discussions

Reply:
but modding has been called such :thinking: 
also "forum watchlist" sounds more like watchlist for forum instead of topic